Spigot-API-Patches/0039-LootTable-API.patch

```diff
@@ -0,0 +1,260 @@
+From 273aa76fce2f98526420e9145aa5287f5b1034f4 Mon Sep 17 00:00:00 2001
+From: Aikar <aikar@aikar.co>
+Date: Sun, 1 May 2016 15:19:49 -0400
+Subject: [PATCH] LootTable API
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/loottable/Lootable.java b/src/main/java/com/destroystokyo/paper/loottable/Lootable.java
+new file mode 100644
+index 0000000..6cbc765
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/loottable/Lootable.java
+@@ -0,0 +1,33 @@
++package com.destroystokyo.paper.loottable;
++
++import org.bukkit.World;
++
++public interface Lootable {
++
++    /**
++     * Gets the name of the Loot Table to be used in the World Folder
++     * @return The name, or null if no loot table exists
++     */
++    String getLootTableName();
++
++    /**
++     * Sets the name of the Loot Table to be used in the World Folder
++     * Will use a random seed
++     *
++     * @param name name in either foo or minecraft:foo format
++     * @return The previous Loot Table before the change
++     */
++    default String setLootTableName(String name) {
```

Comment:
I can see changing setter, but getLootTable() is not valid. Because that can be used to return an actual LootTable object later down the road, and were only returning the name of the table.